tests/jerry/es2015/json-stringify.js

```diff
@@ -0,0 +1,17 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Test with proxy
+assert(JSON.stringify(new Proxy(['foo'], {})) === '["foo"]');
+assert(JSON.stringify(new Proxy({0:"foo"}, {})) === '{"0":"foo"}');
```

Comment:
Please add a tc, when the Proxy handler get accessor throws an error whenever the "length" property is accessed.